mqtt-sn-2.0/prose/editorial-content-timetravel/content-autobahn.md

```diff
@@ -1983,6 +1983,8 @@ Situations in which the Will Message is published include, but are not limited t
 
 -   The Client fails to communicate within the Keep Alive time.
 
+-   The Server closes the Network Connection because of a protocol error.
```

Comment:
The single real change - the rest is mainly noise from the proprietary image handling behind the scenes ...